### ISSUE TITLE
Opposite of what this tool does?

### ISSUE THREAD

**Issue by janpio** [emoacht/HelloSwitcher#11]
Hey, I am having the _opposite_ problem of what this tool used to solve: I attached an external camera, and with the recent Win11 updates now this camera is always used for Hello facial recognition. The problem is that because of the positioning of that camera, it almost never works. I would very much prefer to just use my laptop's (Surface Laptop Studio) built in camera as that is much easier and reliable for me to use and unlock Windows. Any ideas?

**Comment by emoacht** [comment]
If you disable the external camera, the internal built-in camera will be used for Windows Hello.
But I guess you want to keep the external camera's functionaly as a ordinary camera (for visible rigjt not IR) and I have no idea how to disable the functionality for Windows Hello only.

**Comment by emoacht** [comment]
If you disable the external camera, the internal built-in camera will be used for Windows Hello.
But I guess you want to keep the external camera's functionaly as a ordinary camera (for visible right not for IR) and I have no idea how to disable the functionality for Windows Hello only.

**Comment by janpio** [comment]
Exactly, that was my plan - use the laptop IR for login, but use the 4K external camera for Zoom etc. 

I thought I could maybe disable the "IR device" of my Logitech BRIO manually, but no luck in device manager (although finding anything there is close to impossible). Also no setting in Windows (which would be natural...) to choose which camera it should use for login.
